packages/jest/README.md

```diff
@@ -7,93 +8,107 @@ Allows running `ng test` with Jest instead of Karma & Jasmine.
 The builder comes to provide zero configuration setup for Jest while keeping the workspace clear of boilerplate code.
 
 ## Prerequisites
-  - [Angular CLI 8](https://www.npmjs.com/package/@angular/cli)
-  - [Jest 24](https://www.npmjs.com/package/jest)
-  
+
+- [Angular CLI 8](https://www.npmjs.com/package/@angular/cli)
+- [Jest 24](https://www.npmjs.com/package/jest)
+
 ## Installation
+
 1. Remove Karma related libraries and files:
-   ```Shell
-	npm remove karma karma-chrome-launcher karma-coverage-istanbul-reporter karma-jasmine karma-jasmine-html-reporter
-	rm ./karma.conf.js ./src/test.ts
+   ```console
+   npm remove karma karma-chrome-launcher karma-coverage-istanbul-reporter karma-jasmine karma-jasmine-html-reporter
+   rm ./karma.conf.js ./src/test.ts
    ```
 2. Install the builder (and `jest` if you still haven't): `npm i -D jest @types/jest @angular-builders/jest`
 
 ## Updating Typescript configurations
-1. In _tsconfig.spec.json_ (root directory, used by Jest): 
+
+1. In _tsconfig.spec.json_ (root directory, used by Jest):
+
    - Replace `jasmine` in `types` array with `jest`  
      _You want your tests to be type-checked against Jest typings and not Jasmine._
    - Remove `test.ts` entry from `files` array  
      _This file was responsible for Karma setup, you don't need it here anymore._
 
-2. In  _tsconfig.json_ (root directory, used by IDE): 
+2. In _tsconfig.json_ (root directory, used by IDE):
    - Add `jest` to `types` array  
-     _Although you run your unit tests with Jest, Protractor (e2e tests) still [has to use Jasmine](https://github.com/angular/protractor/issues/3889). Due to this fact it’s possible that you favorite IDE will get confused with the typings and will propose you Jasmine types in unit tests.  
-    `tsconfig.json` is the config file that your IDE uses so you have to instruct it explicitly to use Jest typings.  
-     Bear in mind that the other side of the coin is that your IDE will propose you Jest types in your e2e tests._
+      _Although if you run your unit tests with Jest, Protractor (e2e tests) still [has to use Jasmine](https://github.com/angular/protractor/issues/3889). Due to this fact it’s possible that you favorite IDE will get confused with the typings and will propose you Jasmine types in unit tests.  
```

Comment:
The meaning is that "Even though YOU run your unit tests with Jest, Protractors still has to use Jasmine". So `if` is inappropriate here.

Reply:
Isn't it clear enough? You think we need to rephrase it?

Reply:
I didn’t understand it correctly, I will revert :D Np